guard/resources/parse-tree/output-dir/test_rule_iterate_through_json_list_without_key.out

```diff
@@ -0,0 +1,145 @@
+assignments:
```

Comment:
I think for known formats of output, we can keep the extension as the format. So this and next file could be called `whatever.yaml` - it will show us text highlighting in IDE. Only for outputs that aren't structured in a single format we can leave the wildcard `.out` to be used.